### ISSUE TITLE
Symbol pd_commnand_callback_t contains spelling typo

### ISSUE THREAD

**Issue by pcpro178** [goToMain/libosdp#83]
Symbol pd_commnand_callback_t needs to be renamed to pd_command_callback_t to fix spelling typo.

**Comment by pcpro178** [comment]
I already have a fix for this, but I'm unable to assign myself to the ticket.  When I try to push the branch, it gives me a permission denied error.

```
$ git push --set-upstream origin symbol_typo_fix
remote: Permission to goToMain/libosdp.git denied to pcpro178.
fatal: unable to access 'https://github.com/goToMain/libosdp/': The requested URL returned error: 403
```

**Comment by omnisci3nce** [comment]
Hah, I was about to point this out too.

**Comment by sidcha** [comment]
I'm going to fix this myself and add you as the reporter.

**Comment by sidcha** [comment]
Fixed in a7913ba5
